src/GitHub.Exports/Services/IRepositoryFacade.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LibGit2Sharp;
+
+namespace GitHub.Services
+{
+    /// <summary>
+    /// Facade for <see cref="LibGit2Sharp.Repository"/> static methods.
+    /// </summary>
+    public interface IRepositoryFacade
```

Comment:
Do we need a new interface for these? Would they make sense in `IGitService`?

Reply:
That was actually the problem I had. The methods I needed to mock were in `IGitService`. I wanted a pure facade for the static methods in `LibGit2Sharp.Repository`. I've hit the problem before, so thought having a `IRepositoryFacade` interface/service would be useful in other places.

Reply:
Oh, I understand: you mean the methods you had to _test_ are in `IGitService`. Ok yeah I missed that, LGTM ;)